### PR TITLE
Prepare release r1.1

### PR DESCRIPTION
**Pull request by jlurien**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:

Make final updates required for r1.1

- Adjust versions in yaml/feature for -rc.1
- Update API readiness templates
- README
- CHANGELOG

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for reviewers:

- For location-verification some final updates to user stories and external documentation (implementation guidelines) is tbd prior to M4
- Test result statement  also tbd (not clear what to provide in link)

### CODE REVIEW

**Review by maxl2287 — APPROVED** [review]
LGTM, just added an additional `Add` for geofencing Changelog

**Review by maxl2287 — APPROVED** [review]

**Review by hdamker — COMMENTED** [review]
Looks good from Release Management perspective - just one typo found and one proposal

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,88 @@
...
 
 **Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**
 
+The below sections record the changes for each API version in each release as follows:
+
+* for each first alpha or release-candidate API version, all changes since the release of the previous public API version
+* for subsequent alpha or release-candidate API versions, the delta with respect to the previous pre-release
+* for a public API version, the consolidated changes since the release of the previous public API version
+
+# r1.1m
```
[the remark this answers is not in the document]
Comment (jlurien) [reply CHANGELOG.md:17]: typo

**Review by maxl2287 — APPROVED** [review]

**Comment by hdamker** [comment]
@jlurien Ready to go 🚀

**Pull request merged**